website/docs/api/actions/drivers/driver-actions.md

```diff
@@ -25,6 +25,17 @@ import static io.github.boykaframework.actions.drivers.DriverActions.withDriver;
 String output = withDriver ().executeScript ("alert('Hello World');");
 ```
 
+### `pressKeys (keys)` {#press-keys}
+
+ This method will simulate the given keys being pressed.
+
+ ```java
+import static io.github.boykaframework.actions.drivers.DriverActions.withDriver;
+import org.openqa.selenium.Keys;
+
+withDriver ().pressKeys (Keys.ENTER, "Hello World");
```

Comment:
what is the second parameter("Hello World") here? Will it type the text "Hello World" using the keys?
If yes, then show different examples, one using Keys.ENTER, another with showing only "Hello World" and third one using both, Keys. ENTER, "Hello Understand". This will help user to understand better